README.md

```diff
@@ -56,24 +56,43 @@ and [contributing guidelines](CONTRIBUTING.md).
 
 ## Quickstart
 
-Run this command to start FerretDB with PostgreSQL:
+Run this command to start FerretDB with PostgreSQL, ensure to update `<username>` and `<password>`:
```

Comment:
Looks like you now have the same style as @Fashander 😆 

The use of the comma between the two imperatives (“Run” and “ensure”) is a little awkward. It also implies a sequence of commands ("run" first, "ensure" second), but that's not what we mean.

The most natural phrasing is probably "making sure".

Reply:
Probably comes from keeping the consistency with existing usage :see_no_evil:

Reply:
> The use of the comma between the two imperatives (“Run” and “ensure”) is a little awkward. It also implies a sequence of commands ("run" first, "ensure" second), but that's not what we mean.

That all still applies to "run" and "make"